src/main/java/de/tum/in/www1/artemis/config/SecurityConfiguration.java

```diff
@@ -216,9 +215,10 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                 }
             )
             // Applies additional configurations defined in a custom security configurer adapter.
-            .with(securityConfigurerAdapter(), configurer -> configurer.configure(http))
+            .with(securityConfigurerAdapter(), configurer -> configurer.configure(http));
             // Enable HTTP Basic authentication so that people can authenticate using username and password against the server's REST API
-            .httpBasic(Customizer.withDefaults());
+            // FIXME: This breaks LocalCI buildagents
+            //.httpBasic(Customizer.withDefaults());
```

Comment:
Is something like this possible=
```suggestion
            if (!profileService.isBuildAgent()) {
               http.httpBasic(Customizer.withDefaults());
            }
```

Reply:
Sadly no. The issue here is that build agents do not have LocalVC enabled at all, so they need to call a core node for the repository. 
And if the core node has http basic auth enabled, the build agent is unable to clone because of an authorization issue.